### ISSUE TITLE
./wakeWordAgent -e sensory no such file or directory (same error on kitt one too)

### ISSUE THREAD

**Issue by Invincer** [alexa/alexa-avs-sample-app#449]
Everything seemed to go well on the install until terminal window #3 at the end dealing with the wakeword. The instructions said

To use the Sensory wake word engine, type -

cd ~/Desktop/alexa-avs-sample-app/samples
cd wakeWordAgent/src && ./wakeWordAgent -e sensory
or, type this to use KITT.AI's wake word engine -

cd ~/Desktop/alexa-avs-sample-app/samples
cd wakeWordAgent/src && ./wakeWordAgent -e kitt_ai
but then I get the"No such file or directory"

Based on some issues others had, I tried going into the wakeword/src directory and doing "cmake .", which seemed to work. But then when I tried "make", I get the following error...

~/Desktop/alexa-avs-sample-app/samples/wakeWordAgent/src $ make [ 6%] Building CXX object CMakeFiles/wakeWordAgent.dir/GPIOWakeWordEngine.cpp.o /home/pi/Desktop/alexa-avs-sample-app/samples/wakeWordAgent/src/GPIOWakeWordEngine.cpp:6:22: fatal error: wiringPi.h: No such file or directory #include <wiringPi.h> ^ compilation terminated. CMakeFiles/wakeWordAgent.dir/build.make:374: recipe for target 'CMakeFiles/wakeWordAgent.dir/GPIOWakeWordEngine.cpp.o' failed make[2]: *** [CMakeFiles/wakeWordAgent.dir/GPIOWakeWordEngine.cpp.o] Error 1 CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/wakeWordAgent.dir/all' failed make[1]: *** [CMakeFiles/wakeWordAgent.dir/all] Error 2 Makefile:83: recipe for target 'all' failed make: *** [all] Error 2

Any help would be greatly appreciated, this is my first real pi project and I thought it was going well there till the end.

**Comment by elstryan** [comment]
Hi, 
Can you please confirm that you are running this on a Raspberry-Pi? And if you are which kind of Pi? 

The header that your build is looking for should be included in the /usr/include/ directory on all RaspberryPis. Can you check that directory and see if the file exists?

**Comment by guidoffm** [comment]
I have had this problem too. I had to restart the build of the wakeword engine again and after this everthing was OK.

**Comment by Invincer** [comment]
I am on Pi 3. Got it working, had to install wiringpi at http://wiringpi.com/download-and-install/, then cmake. in the wakewordagent dir, then make

**Comment by tscott16** [comment]
I had the same issue with both sensory and kitt_ai resulting in "no such file or directory" in the 3rd Terminal window. I tried re-running the automated_install.sh given in step 5 in the guide but to no avail. The install script did show some errors with wiringPi so followed the guide on [http://wiringpi.com/download-and-install/](url) starting at: 

`$ sudo apt-get install git-core`
 
Once I'd installed wiringPi, I re-ran the automated_install.sh and all went perfectly. I then went through the 3 terminal windows again and wakeWordAgent worked first time. 

As I'm from the UK, the companion app was showing my RPi device as offline and was preventing from creating skills etc., I downloaded the US apk and this sorted the issue.
